tests/cli/fvm_smoke_tests/fvm_basic.bats

```diff
@@ -809,3 +809,41 @@ setup_file() {
     rm -rf $FLUVIO_HOME_DIR
     assert_success
 }
+
+@test "Disallows switching versions when there are active proccesses" {
+    run bash -c '$FVM_BIN self install'
+    assert_success
+
+    # Sets `fvm` in the PATH using the "env" file included in the installation
+    source ~/.fvm/env
+
+    # Installs Fluvio Stable
+    run bash -c 'fvm install'
+    assert_success
+
+    # Starts Fluvio Cluster
+    run bash -c 'fluvio cluster start --local' &
+    sleep 10
+    assert_success
```

Comment:
The `&` at the end is required so the process is spawned and the test continues in parallel. Then add some sleep because we are not hooked to the process anymore and starting the cluster takes some time.

Reply:
Why can't we wait until the process 'fluvio cluster start --local' finishes? It takes a couple of seconds to start and then this process ends. If it hangs we should find out why. Relying on `sleep 10` is very flaky, especially on slow CI execution sometimes.

Reply:
Yeah it hangen in my local but I can try here!

I agree with the `sleep` comment!

Reply:
Another way to do it instead of a long sleep is to loop on an operation to verify response (e.g. `fluvio topic list`) with a short sleep

Reply:
Thats a good one! Im not sure if you already saw the cluster deletion error in CI?

Im investigating on it, because for this test suite we shouldnt have any Clusters running, but turns out, that it seems to be a Cluster and when I run:

```bash
fluvio cluster delete
```

It is failing with _Failed to delete Helm Charts_.

Perhaps the loop you mention could be helpful, to "force" deletion. But I want to make sure no cluster is present first.